build_rln.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+# This script is used to build the rln library for the current platform, or download it from the
+# release page if it is available.
+
+set -e
+
+# first argument is the build directory
+build_dir=$1
+
+if [ -z "$build_dir" ]; then
```

Comment:
If we are using bash just use `[[ ]]` for `if` clauses. You should probably just run `shellcheck` on the whole script.

Reply:
Addressed in [2ea1ca0](https://github.com/waku-org/nwaku/pull/1603/commits/2ea1ca007a0923c8cd07f85ada244a1ee07cf1f6)